src/app/api/routes/devices.py

```diff
@@ -39,3 +37,15 @@ async def delete_device(device_id: int = Path(..., gt=0), _=Security(get_current
 @router.get("/my-devices", response_model=List[DeviceOut])
 async def fetch_my_devices(me: UserRead = Security(get_current_user, scopes=["me"])):
     return await routing.fetch_entries(devices, ("owner_id", me.id))
+
+
+@router.post("/heartbeat", response_model=HeartbeatOut)
+async def heartbeat(device: DeviceOut = Security(get_current_device, scopes=["device"])):
+    return await crud.DeviceCRUD.heartbeat(device)
+
+
+@router.post("/{device_id}/update_location", response_model=DeviceOut)
```

Comment:
happy to go for "_" separators rather than "-" in route naming, but perhaps it would be best to keep the same convention all of them?

Reply:
Done